### ISSUE TITLE
"Returns" clause in numpy docstring

### ISSUE THREAD

**Issue by aldanor** [KristoforMaynard/SublimeAutoDocstring#5]
The snippet currently generates this for `numpy`-style docstrings:

```
Returns
-------
TYPE : Description
```

However, it's not entirely correct, according to https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt:

>    Explanation of the returned values and their types. Similar to the
>    **Parameters** section, except the name of each return value is optional.
>    The type of each return value is always required::

```
 Returns
 -------
 int
     Description of anonymous integer return value.
```

>    If both the name and type are specified, the **Returns** section takes the
>    same form as the **Parameters** section::

```
 Returns
 -------
 err_code : int
     Non-zero value indicates error code, or zero on success.
 err_msg : str or None
     Human readable error message, or None on success.
```

So it's probably best to do the same thing as in Parameters section, with three placeholders:

```
Returns
----------
name : TYPE
    Description
```

**Comment by KristoforMaynard** [comment]
Good catch. I don't regularly use the numpy style myself, so things like this sneak right past me. I think there should be an option for whether or not in include names with the returns by default, but either way the current behavior is improper style. I'll give this a deeper look when I get a chance.

**Comment by KristoforMaynard** [comment]
This is now fixed in version 0.3.2 with the `default_return_name` option. If that option is given, the docstring will look like:

```
Returns
-------
name : TYPE
    Description
```

and if it is an empty string, then the result is:

```
 Returns
 -------
 TYPE
     Description
```

**Comment by aldanor** [comment]
Neat. Thanks!
